mdetsims/metacal/metacal_fitter.py

```diff
@@ -0,0 +1,284 @@
+import logging
+import numpy as np
+import esutil as eu
+
+from ngmix.bootstrap import MaxMetacalBootstrapper
+from ngmix.gexceptions import BootPSFFailure, BootGalFailure
+
+from .base_fitter import FitterBase, _fit_one_psf
+from .util import Namer
+
+logger = logging.getLogger(__name__)
+
+METACAL_TYPES = ['noshear', '1p', '1m', '2p', '2m']
+
+
+class MetacalFitter(FitterBase):
+    """Run metacal on all a list of observations.
+
+    Parameters
+    ----------
+    conf : dict
+        A configuration dictionary.
+    nband : int
+        The number of bands.
+    rng : np.random.RandomState
+        An RNG instance.
+    mof_fitter : mdetsims.metcal.MOFFitter
+        A an instantiated MOFFitter for doing neighbor subtraction.
+
+    Methods
+    -------
+    go(mbobs_list_input)
+    """
+    def __init__(self, conf, nband, rng, mof_fitter=None, **kw):
+
+        self.mof_fitter = mof_fitter
+
+        super().__init__(conf, nband, rng)
+
+    def _setup(self):
+        self.metacal_prior = self._get_prior(self['metacal'])
```

Comment:
The metacal prior is now built using the code from fitvd, not dbsim.

Reply:
that is fine